.github/workflows/check-semver.yml

```
@@ -39,7 +40,8 @@ jobs:
           export CARGO_TARGET_DIR=target
           export RUSTFLAGS='-A warnings -A missing_docs'
           export SKIP_WASM_BUILD=1
-          if ! parity-publish --color always prdoc --since old --validate prdoc/pr_$PR.prdoc -v; then
+          toolchain=$(parity-publish semver --minimum-nightly-rust-version)
+          if ! parity-publish --color always prdoc --since old --validate prdoc/pr_$PR.prdoc -v --toolchain nightly-2024-03-01; then
```

Was it supposed to be
```suggestion
          if ! parity-publish --color always prdoc --since old --validate prdoc/pr_$PR.prdoc -v --toolchain $toolchain; then
```
?